src/internal/components/abstract-switch/styles.scss

```diff
@@ -65,25 +79,10 @@
   }
 }
 
-.control {
-  position: relative;
-  /* stylelint-disable selector-max-type */
-  & > input,
-  & > svg,
-  & > .outline {
-    position: absolute;
-    inline-size: 100%;
-    block-size: 100%;
-    inset-block-start: 0;
-    inset-inline-start: 0;
-  }
-  & > input {
-    opacity: 0;
-    z-index: 1;
-    user-select: none;
-    cursor: default;
-    margin-block: 0;
-    margin-inline: 0;
+.native-input {
+  &:focus-visible {
+    & + .outline {
+      display: block;
+    }
```

Comment:
The reordering here is to satisfy stylelint's `no-descending-specificity` rule. Without the order change, stylelint complains:

```
Expected selector "& > .outline" to come before 
     selector ".native-input:focus-visible + .outline"  
     no-descending-specificity
```

Reply:
On second thought, I've restored the original ordering here to prevent possible changes in behavior.